<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IlsLlzPanel</class>
 <widget class="QWidget" name="IlsLlzPanel">
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>260</width>
    <height>140</height>
   </size>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="topMargin">
    <number>4</number>
   </property>
   <property name="bottomMargin">
    <number>4</number>
   </property>
   <property name="leftMargin">
    <number>4</number>
   </property>
   <property name="rightMargin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="lblTitle">
     <property name="text">
      <string>ILS/LLZ BRA</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignHCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="grpFacility">
     <property name="title">
      <string>Directional Facility</string>
     </property>
     <layout class="QFormLayout" name="formFacility">
      <item row="0" column="0">
       <widget class="QLabel" name="lblFacility">
        <property name="text">
         <string>Type</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboFacility"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
   <widget class="QGroupBox" name="grpInputLayers">
     <property name="title">
      <string>Input Layers</string>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QFormLayout" name="formInputLayers">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblNavaidLayer">
        <property name="text">
         <string>Navaid layer (point)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="cboNavaidLayer"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblRoutingLayer">
        <property name="text">
         <string>Routing / runway (line)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="cboRoutingLayer"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
   <widget class="QGroupBox" name="grpParameters">
     <property name="title">
      <string>Parameters</string>
     </property>
     <layout class="QFormLayout" name="formParameters">
      <property name="horizontalSpacing">
       <number>4</number>
      </property>
      <property name="verticalSpacing">
       <number>4</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lblSiteElev">
        <property name="text">
         <string>Site elevation (m)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spnSiteElev">
        <property name="minimum">
         <double>-1000.000000000000000</double>
        </property>
        <property name="maximum">
         <double>10000.000000000000000</double>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblA">
        <property name="text">
         <string>A (m)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spnA">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>100000.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>0.000000000000000</double></property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lblB">
        <property name="text"><string>b (m)</string></property>
       </widget>
      </item>
      <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spnB">
       <property name="minimum"><double>0.000000000000000</double></property>
       <property name="maximum"><double>100000.000000000000000</double></property>
       <property name="decimals"><number>2</number></property>
       <property name="value"><double>0.000000000000000</double></property>
      </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lblh">
        <property name="text"><string>h (m)</string></property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spnh">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>100000.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>70.000000000000000</double></property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lblr">
        <property name="text"><string>r (m)</string></property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spnr">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>100000.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>6000.000000000000000</double></property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="lblD">
        <property name="text"><string>D (m)</string></property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="spnD">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>100000.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>600.000000000000000</double></property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="lblH">
        <property name="text"><string>H (m)</string></property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="spnH">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>100000.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>20.000000000000000</double></property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="lblL">
        <property name="text"><string>L (m)</string></property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QDoubleSpinBox" name="spnL">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>100000.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>1500.000000000000000</double></property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="lblPhi">
        <property name="text"><string>Ï• (deg)</string></property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="spnPhi">
        <property name="minimum"><double>0.000000000000000</double></property>
        <property name="maximum"><double>90.000000000000000</double></property>
        <property name="decimals"><number>2</number></property>
        <property name="value"><double>40.000000000000000</double></property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="lblOutputName">
        <property name="text"><string>Output name (optional)</string></property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="txtOutputName">
        <property name="placeholderText">
         <string>e.g. RWY01 or custom label</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="layoutButtons">
     <property name="spacing">
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="btnDirection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Direction: Start to End</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCalculate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Calculate</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnClose">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
     <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
            <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
         </property>
        </spacer>
     </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>